internal/provider/utils/noportforward.go

```diff
@@ -37,3 +40,7 @@ func (n *NoPortForwarding) KeepPortForward(context.Context, uint16, netip.Addr,
 	string, Logger) (err error) {
 	return fmt.Errorf("%w: for %s", ErrPortForwardingNotSupported, n.providerName)
 }
+
+type NoWireguardConfigurator interface {
+	GetWireguardConnection(ctx context.Context, connection models.Connection, wireguardSettings settings.Wireguard, ipv6Supported bool) (settings wireguard.Settings, err error)
+}
```

Comment:
You should implement it like NoPortForwarding instead of just having an interface